qiskit/circuit/quantumcircuit.py

```diff
@@ -2366,6 +2366,11 @@ def cz(self, control_qubit, target_qubit,  # pylint: disable=invalid-name
         return self.append(CZGate(label=label, ctrl_state=ctrl_state),
                            [control_qubit, target_qubit], [])
 
+    def pauli(self, qubits, pauli_string):  # pylint: disable=invalid-name
```

Comment:
For other parameterized gates the params goes before the qubits, so this should be 
```suggestion
    def pauli(self, label, qubits):  # pylint: disable=invalid-name
```